scripts/src/blockchain/transactionBuilder.ts

```diff
@@ -35,8 +44,12 @@ export class TransactionBuilder {
 		return this;
 	}
 
+	public set channel(channle: Channel | undefined) {
+		(this as any)._transactionBuilder._channel = channle;
+	}
+
 	public get channel(): Channel | undefined {
-		return this._channel;
+		return (this as any)._transactionBuilder._channel;
```

Comment:
any reason for this change? the channel doesn't exist on the base transaction builder.